backend/src/libs/sequelize.js

```diff
@@ -1,7 +1,7 @@
 const Sequelize = require('sequelize');
 const log         = require('./log')(module);
 
-const sequelize = new Sequelize('postgres://vagrant:vagrant@127.0.0.1:5432/pkeeper');
+const sequelize = new Sequelize('postgres://vagrant:vagrant@postgres:5432/pkeeper');
```

Comment:
It will break vagrant, isn't it. Is there an option to keep both? If not, just delete everything related to vagrant.